transformers/uws/server.js

```diff
@@ -69,6 +69,8 @@ module.exports = function server() {
     spark.emit('incoming::data', msg);
   });
 
+  native.server.group.onPing(group, (msg, spark) => spark.heartbeat());
```

Comment:
Time flies. That was actually the two of us... completely forgot. https://github.com/primus/primus/issues/363